config/schema.error.yaml

```
@@ -0,0 +1,575 @@
+    step: compile-di
+    title: 'Command `/bin/magento setup:di:compile` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: build
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::BUILD_COMPOSER_DUMP_AUTOLOAD_FAILED:
+    step: dump-autoload
+    title: 'Command `composer dump-autoload` failed'
+    suggestion: 'The `composer dump-autoload` command failed. Check the `cloud.log` for more information.'
+    stage: build
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::BUILD_BALER_NOT_FOUND:
+    step: run-baler
+    title: 'The command to run `Baler` for Javascript bundling failed'
+    suggestion: 'Check the `SCD_USE_BALER` environment variable to verify that the Baler module is configured and enabled for JS bundling. If you don''t need the Baler module, set `SCD_USE_BALER: false`.'
+    stage: build
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::BUILD_UTILITY_NOT_FOUND:
+    step: compress-static-content
+    title: 'Required utility wasn''t found (timeout, bash)'
+    suggestion: ''
+    stage: build
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::BUILD_SCD_FAILED:
+    step: deploy-static-content
+    title: 'Command `/bin/magento setup:static-content:deploy` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: build
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::BUILD_SCD_COMPRESSION_FAILED:
+    step: compress-static-content
+    title: 'Static content compression failed'
+    suggestion: 'Check the `cloud.log` for more information.'
+    stage: build
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::BUILD_SCD_COPYING_FAILED:
+    step: 'backup-data: static-content'
+    title: 'Failed to copy static content into the `init` directory'
+    suggestion: 'Check the `cloud.log` for more information.'
+    stage: build
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::BUILD_WRITABLE_DIRECTORY_COPYING_FAILED:
+    step: 'backup-data: writable-dirs'
+    title: 'Failed to copy some writable directories into the `init` directory'
+    suggestion: 'Failed to copy writable directories into the `./init` folder. Check your filesystem permissions.'
+    stage: build
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::BUILD_UNABLE_TO_CREATE_LOGGER:
+    title: 'Unable to create a logger object'
+    suggestion: ''
+    stage: build
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::BUILD_CLEAN_INIT_PUB_STATIC_FAILED:
+    step: 'backup-data: static-content'
+    title: 'Failed to clean the `./init/pub/static/` directory'
+    suggestion: 'Failed to clean `./init/pub/static` folder. Check your filesystem permissions.'
```

```suggestion
    suggestion: 'Failed to clean `./init/pub/static` folder. Check your filesystem permissions.'
```

General observation:
If the title matches the first sentence of the suggestion, do you need to include the title?  If not, suggest changing like this:

```    title: 'Failed to clean the `./init/pub/static/` directory'
    suggestion: Check your filesystem permissions.'
```

---

For many errors, the title will be replaced with dynamic (more informative) error message from the script. 
This title will be used only in case of running the CLI command for error describing.